pkg/common/args_test.go

```diff
@@ -35,7 +36,7 @@ func TestValidateArgs(t *testing.T) {
 		skip          string
 		expectedSkip  string
 		extraArgs     []string
```

Comment:
We can remove these fields as we won't be using them.

Reply:
added it back, had removed it by mistake

Reply:
We are not really modifying the value of `skip` inside `ValidateArgs()` instead just printing it out, thus it will be better to remove the test for skip.